docs/setup/using_in_browser.md

```diff
@@ -0,0 +1,10 @@
+[Initialization of library](./intialization.md) requires several properties of [signer](../api/classes/modules_signer.SignerService.md):
+
+- The signer meets [EIP-191: Signed Data Standard](https://eips.ethereum.org/EIPS/eip-191)
+- Public key. This is legacy parameter, which was used to initialized DID document
+
+These properties are not directly exposed by all signers, such as Metamask, and thus are determined in runtime by sending test sign request.
+In order to improve user expirience when library is used in browser it tries to get these parameters from local storage.
+In order to implement application specific session management the library doesn't modify storage and rely on the application to set `isEthSigner` and `PublicKey` local storage variables on login.
```

Comment:
```suggestion
In order to implement application specific session management the library doesn't modify storage and relies on the application to set `isEthSigner` and `PublicKey` local storage variables on login.
```

Reply:
I don't rely understand this logic. We want "application specific session management" so "the library doesn't modify storage", but the library is relying on "local storage variables".
So we don't allow the library to write to local storage but the library relies on local storage, so it seems the use of local storage for session management is forced by the library. Maybe you could add an example of how "application specific session management" would be used?

Reply:
Library doesn't force use of local storage. Application can refuse from session at all and library will calculate signer properties on each login